### DISCUSSION TITLE
"randomness" in the competition

### DISCUSSION THREAD

**Discussion by TomatoPicker278** [empriselab/RCareWorld#150] [Other Q&A] [answered]
Hi!
Regarding the randomness in the bathing scene, I noticed on the competition website it states:

"——What aspects of each task are randomized?
——For bathing: the human's limb poses, the initial position of the sponge, and the initial position of the robot's base."

I would like to ask if the "position of the sponge and the robot's base" here refers only to the 3D position and not the orientation in 3D space? More specifically, in the final test of the first phase, will the orientation of the sponge and the robot always be fixed, with only their spatial positions changing?

**Comment by YoruCathy** [comment, marked as answer]
Yes, the initial position refers to position, not orientation.

**Reply by TomatoPicker278** [reply]
Thanks!
